config/set/laminas-servicemanager-40.php

```diff
@@ -2,6 +2,7 @@
 
 declare(strict_types=1);
 
+use Rector\LaminasServiceManager4\Rector\MethodCall\RemoveServiceLocatorRector;
```

Comment:
The namespace should be `Lamaminas\\`, until we support many various version and rules for the framework, like only Symfony/Laravel have now.

Reply:
updated.